content/references/protocol/transactions/types/payment.md

```diff
@@ -41,13 +41,14 @@ Payments are also the only way to [create accounts](#creating-accounts).
 
 | Field            | JSON Type            | [Internal Type][] | Description    |
 |:-----------------|:---------------------|:------------------|:---------------|
-| `Amount`         | [Currency Amount][]  | Amount            | The amount of currency to deliver. For non-XRP amounts, the nested field names MUST be lower-case. If the [`tfPartialPayment` flag](#payment-flags) is set, deliver _up to_ this amount instead. |
+| `Amount`         | [Currency Amount][]  | Amount            | [API v1][]: The maximum amount of currency to deliver. For non-XRP amounts, the nested field names MUST be lower-case. If the [`tfPartialPayment` flag](#payment-flags) is set, deliver _up to_ this amount instead.<br>[API v2][]: Field renamed to `DeliverMax`. |
```

Comment:
It might be better to have a separate table row for `DeliverMax` and have the description for each explain (API v1/2 only, alias for the field in other API version)